pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -752,38 +760,6 @@ func updateLispConfiglets(ctx *zedrouterContext, legacyDataPlane bool) {
 // Wrappers around handleCreate, handleModify, and handleDelete
 
 // Determine whether it is an create or modify
```

Comment:
Shouldn't this comment be removed since the function to which it refers has been removed?